spec/controllers/admin_users_account_anonymising_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe AdminUsersAccountAnonymisingController do
+
```

Comment:
The one style was to pad `describe`/`context` blocks with extra spaces - we're not doing that anymore. 
```suggestion
```